src/renderer/angular/services/ledger.service.js

```diff
@@ -0,0 +1,88 @@
+'use strict';
+const { Logger } = require('logger/logger');
+const log = new Logger('LedgerService');
+const EthUnits = require('../classes/eth-units');
+const async = require('async');
+
+function LedgerService(RPCService, CommonService, Web3Service) {
+	'ngInject';
+
+	log.info('LedgerService Initialized');
+
+	class LedgerService {
```

Comment:
We should try to move all of this device interaction logic out of the renderer and into the node process in the future. Having it tightly coupled to Angular and the presentation layer makes it hard to re-use for cases like the browser extension, where we might want to trigger transaction signing flows remotely.

Reply:
it should not be here, probably a merge error. Fixing